cmd/litestream/main.go

```diff
@@ -176,8 +176,8 @@ type Config struct {
 	Logging LoggingConfig `yaml:"logging"`
 
 	// MCP server options
-	MCPEnabled bool `yaml:"mcp-enabled"`
-	MCPPort    int  `yaml:"mcp-port"`
+	MCPEnabled bool   `yaml:"mcp-enabled"`
+	MCPAddr    string `yaml:"mcp-addr"`
```

Comment:
You can drop `MCPEnabled` and just check if `MCPAddr` is non-blank to see if it's enabled. You'll need to take off the default addr though.